xc/xc7/tests/soc/litex/mini_ddr_eth/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+add_file_target(FILE arty_clocks.xdc)
+
+set(SOURCES
+  top.v
+  top.xdc
+  mem.init
+  mem_1.init
+  mem_2.init
+)
+
+add_litex_test(
+  NAME minilitex_ddr_eth_arty
+  LITEX_DIR arty_soc
+  LITEX_BOARD a7-35
+  LITEX_SOURCES ${SOURCES}
+  EXTERNAL_SOURCES ${VEXRISCV}
+  BOARD arty-full
+  GENERATE_SCRIPT ${GENERATE_LITEX}
+  FIXUP_SCRIPT ${FIXUP_XDC}
+  FLAGS
+    --with-ram
+    --with-ethernet
+  USE_XDC
+  VIVADO_XDC arty_clocks.xdc
+)
+
+add_litex_test(
+  NAME minilitex_ddr_eth_arty_100t
+  LITEX_DIR arty_a7_soc
+  LITEX_BOARD a7-100
+  LITEX_SOURCES ${SOURCES}
+  EXTERNAL_SOURCES ${VEXRISCV}
+  BOARD arty100t-full
+  GENERATE_SCRIPT ${GENERATE_LITEX}
+  FIXUP_SCRIPT ${FIXUP_XDC}
+  FLAGS
+    --with-ram
+    --with-ethernet
+  USE_XDC
+  VIVADO_XDC arty_clocks.xdc
+  # FIXME: Certain BRAMs do have issues with the bittools regarding the
```

Comment:
Did you confirm this was the case?

Reply:
I only hypothesized that this was the case.

Reply:
This seems to be the issue indeed. I have used the bitread fix from https://github.com/SymbiFlow/prjxray/pull/1335 locally and the error did not show up anymore.

Reply:
Can you please file an issue to re-enable this DIFF_FASM test once the bittool fix is in?  And link that issue here?